Dockerfile

```diff
@@ -8,7 +8,7 @@ VOLUME /config
 RUN apt-get update \
     && apt-get -y upgrade \
     && apt-get -y install software-properties-common wget git \
-    && add-apt-repository ppa:transmissionbt/ppa \
+#    && add-apt-repository ppa:transmissionbt/ppa \
```

Comment:
Why comment instead of remove? Should we move back to the ppa packages afterwards? Haven't really gotten into the sources for transmission and what versions are available from ubuntu repositories.